.circleci/config.yml

```diff
@@ -0,0 +1,26 @@
+version: 2.1
+
+executors:
+  python:
+    docker:
+      - image: circleci/python:2.7.13
+    working_directory: ~/repo
+
+jobs:
+  test:
+    executor: python
+    steps:
+      - checkout
+      - run: pip install virtualenv
+      - run: virtualenv -p python2.7 --no-site-packages --distribute ~/.virtualenvs/aurproxy
+      - run: source ~/.virtualenvs/aurproxy/bin/activate
+      - run: sudo pip install -r requirements.txt
+      - run: sudo pip install -r requirements_dev.txt
```

Comment:
Aren't these run in different shells? Sourcing on line 16 probably won't change the environment for lines 17 and 18. The fact that you are using `sudo` there is strong evidence that you're not using the virtualenv at all.